include/uk/plat/lcpu.h

```diff
@@ -99,6 +101,9 @@ void ukplat_lcpu_halt_to(__snsec until);
  */
 void ukplat_lcpu_halt_irq(void);
 
+void uk_udelay(__u16 usec);
+void uk_mdelay(__u16 msec);
```

Comment:
Since this is a purely KVM platform internal function, you can remove `uk_` prefix. We will change the build system sooner or later in order to mask platform symbols towards a common API (issue #200).